text/0000-natvis.md

```diff
@@ -310,22 +313,22 @@ natvis = ["a.natvis", "b.natvis"]
 This would generate a call to rustc similar to the following,
 (for simplicity purposes, most of the rustc command line has been removed):
 
-`rustc -C natvis=path/to/file/a.natvis,path/to/file/b.natvis -Z unstable-options`
+`rustc -C natvis=path/to/file/a.natvis -C natvis=path/to/file/b.natvis -Z unstable-options`
 
-`.natvis` files that contain spaces within the path will be quoted to ensure
-the path remains valid when passing command line options to rustc. Since the `-Z`
-option is a comma-separated list, a comma would not be valid within the path
-for a Natvis file.
+`.natvis` files that contain spaces or commas within the path will be quoted to ensure
+the path remains valid when passing command line options to rustc.
 
 The `CrateRoot` type would also need to be updated to account for `.natvis`
-files for crates within the dependency graph. To reflect this a new field,
-`natvis_files: Lazy<[PathBuf]>,` would be added. This will store the list of
-`.natvis` files that were passed to the invocation of rustc for the specific crate.
+files for crates within the dependency graph. To reflect this a new type
+`pub struct NatvisFile` will be created to ensure the contents of a `.natvis`
+file can be serialized and stored in the crate metadata. The `CrateRoot` would
+contian the field, `natvis_files: Lazy<[NatvisFile]>`.
 
 Another change that would need to be made here is to add a new field to the
-`CrateInfo` type, `pub natvis_files: Option<Vec<PathBuf>>`. This will allow the `MsvcLinker`
-type to query the list of Natvis files that exist within the crate dependency graph
-and add the `/NATVIS` linker arg for each `.natvis` file.
+`CrateInfo` type, `pub used_crates_natvis: Option<Vec<PathBuf>>`. This will
```

Comment:
I don't think this is needed anymore? We can just use `natvis_files` directly.

Reply:
We still need this to be able to add the `/NATVIS` linker argument in the debug info. At this stage we do not have access to the `CrateMetadata` or the `TyCtx`. The `CrateInfo` is accessible which is why this field would still be needed.